### ISSUE TITLE
Update to swiper 5

### ISSUE THREAD

**Issue by dottodot** [zefoy/ngx-swiper-wrapper#238]
Hello just wondering if there are any plans to update the dependency to swiper 5

**Comment by sconix** [comment]
Most likely with the next major release once Angular 9 is out.

**Comment by AtanasChachev** [comment]
Yeah I'm asking the same... there's an issue with the virutal `prependSlide()` method in the 4.5.X version, which is fixed in `5.0.3` version.

**Comment by dottodot** [comment]
There is also this issue which has been fixed in the lastest version https://github.com/nolimits4web/swiper/issues/2956

**Comment by AtanasChachev** [comment]
@dottodot I've created a fork and updated the package to use swiper: 5.0.3 version until the official package `ngx-swiper-wrapper` gets update. You can add it in your by typing the following in your project

`npm uninstall ngx-swiper-wrapper --save`
`npm install git://github.com/AtanasChachev/ngx-swiper-wrapper-swiper-5.git#swiper-5 --save`

**Comment by kmturley** [comment]
I needed the latest version to use the new param: `updateOnWindowResize: false`

Created a fork here:
https://github.com/kmturley/ngx-swiper-wrapper

You can install using:
`npm install git+https://github.com/kmturley/ngx-swiper-wrapper.git --save`
